build.rs

```diff
@@ -15,8 +15,11 @@
 //
 
 fn main() -> Result<(), Box<dyn std::error::Error>> {
-    tonic_build::configure().compile(
-        &["./skywalking-data-collect-protocol/language-agent/Tracing.proto"],
+    tonic_build::configure().build_server(false).compile(
+        &[
+            "./skywalking-data-collect-protocol/language-agent/Tracing.proto",
+            "./skywalking-data-collect-protocol/management/Management.proto",
```

Comment:
Why do you need this? I think we don't call management APIs from Rust side.

Reply:
I want to call `reportInstanceProperties` to report the properties such as `os_name`, `host_name`.

Reply:
Then add this when you submit relative codes, please.

Reply:
@jmjoy Could you revert this in this PR?

Reply:
Done.